### PR TITLE
enhance(publish): use note's `desc`field for SEO description

### PR DESCRIPTION
**Pull request by namjul**
docs: 
- [[seo description should be generated from `desc` field of note if available|task.2022.07.17.seo-description-should-be-generated-from-desc-field-of-note-if-available]]
- https://github.com/dendronhq/dendron/issues/1596

# Pull Request Checklist

If you are a community contributor, copy and paste the PR template from [Dendron Community PR Checklist](https://gist.github.com/kevinslin/5d1a6663638259e7dbd33b01975cc00f) and add it to the body of the pull request. 

If you are a team member, copy and paste the template from [Dendron Extended PR Checklist](https://gist.github.com/kevinslin/dfc7a101f21c58478216aa0d70256bc2).

To copy the template, click on the `Raw` button on the gist to copy the plaintext version of the template to this PR.

### CODE REVIEW

File: packages/common-all/src/utils/publishUtils.ts [hunk 1]
```diff
@@ -22,31 +22,25 @@ export class PublishUtils {
...
 
   static getSEOPropsFromNote(note: NoteProps): SEOProps {
-    const { title, created, updated, image } = note;
+    const { title, created, updated, image, desc } = note;
     const { excerpt, canonicalUrl, noindex, canonicalBaseUrl, twitter } =
       note.custom ? note.custom : ({} as any);
     return {
       title,
       excerpt,
+      description: desc || excerpt || undefined,
```
[the remark this answers is not in the document]
Comment (namjul) [reply packages/common-all/src/utils/publishUtils.ts]: [`299cbb9` (#3261)](https://github.com/dendronhq/dendron/pull/3261/commits/299cbb9754f912bec2033c86143e7ec11445a5c7)

**Review by kevinslin — APPROVED** [review]

**Review by bpathakota — CHANGES_REQUESTED** [review]
Would update this to `enhance(publish): use note's desc for SEO description`. For PR titles, these are the [[Steps|dendron://dendron.handbook/area.product.sop.ready-to-ship#steps]] to follow. Let me know if I can clarify anything!

**Comment by namjul** [comment]
> Would update this to `enhance(publish): use note's desc for SEO description`. For PR titles, these are the [[Steps|dendron://dendron.handbook/area.product.sop.ready-to-ship#steps]] to follow. Let me know if I can clarify anything!

Thank you Bindhu. I updated the PR title and used rebase to change the commit messages.

**Comment by bpathakota** [comment]
@namjul The PR should still have the categorization of the PR (e.g. feat/enhance/fix) and based the [[Product Scope|dendron://dendron.dendron-site/dendron.ref.scopes]] chosen should be in parenthesis. e.g. `enhance(publish): use note's desc for SEO description`, similar to this PR - https://github.com/dendronhq/dendron/pull/3237

**Pull request merged**